pkg/clusteroperator/list.go

```diff
@@ -131,3 +131,50 @@ func WaitForAllClusteroperatorsStopProgressing(
 
 	return false, err
 }
+
+// VerifyClusterOperatorsVersion checks if all the clusterOperators have version desiredVersion.
```

Comment:
Can you re-factor and simplify this function?

Reply:
done, thanks